### DISCUSSION TITLE
CD Audio?

### DISCUSSION THREAD

**Discussion by MalvareRJ** [erichelgeson/BlueSCSI-v1#210] [Ideas]
I'm just wondering if there's any chance to have Audio output for an emulated CD-Rom?
I've had success booting an FM Towns computer with specialized software, that redirects a SCSI CD-Rom to appears to the computer's firmware as the internal drive, and that is incredible! The BlueSCSI is amazing! Thank you for this!
The only caveat is that most games use CD Audio, so you can play the games, but completely silent.
Would be great to have Audio output of some sort (I'm aware that the hadware will probably need updates to preamp the audio signals, but that is easy to implement).

Thanks a lot for taking the time to read, and hopefully, considering this request!

**Comment by erichelgeson** [comment]
It's possible, though someone will have to read through the spec and implement it. It's on our radar but no immediate plans to add it.
